cloudflare/resource_cloudflare_logpush_job.go

```diff
@@ -84,12 +85,21 @@ func resourceCloudflareLogpushJobRead(d *schema.ResourceData, meta interface{})
 		return nil
 	}
 
-	d.Set("name", job.Name)
-	d.Set("enabled", job.Enabled)
-	d.Set("logpull_options", job.LogpullOptions)
-	d.Set("destination_conf", job.DestinationConf)
-	d.Set("ownership_challenge", d.Get("ownership_challenge"))
-
+	if err := d.Set("name", job.Name); err != nil {
```

Comment:
I’m not sure whether this is adding any value as `d.Set` is a provider internal method which manages the state and the conditions of which it errors get into some pretty gnarly edge cases that we can’t solve here.